core/src/main/codegen/templates/Parser.jj

```diff
@@ -6316,11 +6328,16 @@ SqlNode BuiltinFunctionCall() :
             if (flag == null) {
                 flag = SqlTrimFunction.Flag.BOTH.symbol(SqlParserPos.ZERO);
             }
+        }
+        [
+            <COMMA> {
+                trimChars = Expression(ExprContext.ACCEPT_SUB_QUERY);
+            }
+        ]
```

Comment:
This comma can be non-optional. Trim of a single argument is already handled in the above case.